packages/renderer/src/lib/compose/ComposeDetails.spec.ts

```diff
@@ -0,0 +1,42 @@
+import '@testing-library/jest-dom';
+import { test, expect, vi } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import ComposeDetails from './ComposeDetails.svelte';
+import { mockBreadcrumb } from '../../stores/breadcrumb';
+
+vi.mock('xterm', () => {
+  return {
+    Terminal: vi.fn().mockReturnValue({ loadAddon: vi.fn(), open: vi.fn(), write: vi.fn(), clear: vi.fn() }),
+  };
+});
+
+beforeAll(() => {
+  (window.events as unknown) = {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    receive: (_channel: string, func: any) => {
+      func();
+    },
+  };
+  (window as any).getConfigurationValue = vi.fn();
+  (window as any).matchMedia = vi.fn().mockReturnValue({
+    addListener: vi.fn(),
+  });
+  (window as any).telemetryPage = vi.fn().mockReturnValue({
+    sendPageView: vi.fn(),
+  });
+  (window as any).ResizeObserver = vi.fn().mockReturnValue({ observe: vi.fn(), unobserve: vi.fn() });
+  (window as any).initializeProvider = vi.fn().mockResolvedValue([]);
+  mockBreadcrumb();
+});
+
+test('Simple test that compose logs are clickable and loadable', async () => {
+  render(ComposeDetails, { composeName: 'foobar', engineId: 'engine' });
+  // Click on the logs href
+  const logsHref = screen.getByRole('link', { name: 'Logs' });
+  fireEvent.click(logsHref);
```

Comment:
Maybe good to add an assertion to check that the content of the log tab is present

Reply:
Done! Added an assertion to check that it's loaded in the test (although it's empty screen).

I'm unsure if we should add a more inclusive test for loading the containers, I was thinking of coordinating with @odockal and getting more of an integration test added since we do not mock xterm